src/app/movies/movies.component.html

```diff
@@ -14,10 +14,13 @@
     </mat-menu>
   </div>
 </div>
-<div class="movie" *ngFor="let movie of movies">{{movie}}
+<!-- <div class="movie" *ngFor="let movie of movies">{{movie}}
+   -->
```

Comment:
coś tu jest nie tak, czemu to jest zakomentowane?

Reply:
a już widze niżej, dżizus, sorki

Reply:
a nie no cuja bo to jest dodanei zakomentowane. Nie zostawiaj takich rzeczy (koemntarzy, logów) - przeglądaj sam sobie pra przed wysłaniem komuś do review żeby nie było takiego śmiecia ;)